src/test/HostActivation.Tests/NativeHosting/HostContext.cs

```diff
@@ -477,12 +465,10 @@ public void CompatibilityCheck_Properties(string scenario, bool hasMultiplePrope
             try
             {
                 result = Command.Create(sharedState.NativeHostPath, args)
-                    .CaptureStdErr()
-                    .CaptureStdOut()
-                    .EnvironmentVariable("COREHOST_TRACE", "1")
+                    .EnableTracingAndCaptureOutputs()
```

Comment:
Since most of the native host tests use the same `NativeHosting.SharedTestStateBase` it should be possible to add one method to create the command and reuse it across all these tests.